packages/leap/pubspec.yaml

```diff
@@ -16,6 +16,13 @@ dependencies:
   pathxp: ^0.4.0
   tiled: ^0.10.1
 
+dependency_overrides:
+  flame_tiled:
+    git:
+      url: https://github.com/flame-engine/flame.git
+      ref: erick.tile-atlas-padding
+      path: packages/flame_tiled
```

Comment:
You probably don't want to commit this 😄

Reply:
Was just for testing ahhahaha, I've bumped to the latest flame_tiled though